app/templates/components/widgets/forms/ticket-input.hbs

```diff
@@ -12,10 +12,10 @@
     {{else if (eq ticket.type 'donation')}}
       <div class="two fields">
         <div class="field">
-          {{input type='number' name='min_price' placeholder=(t 'Min') value=ticket.minOrder}}
+          {{input type='number' name='min_price' onChange=(action 'minMaxCheck' ticket) placeholder=(t 'Min') value=ticket.minPrice min=1}}
```

Comment:
No action needed. Add a computed property on ticket-input component that looks at ticket.minPrice and ticket.minPrice to see if it is valid or not.

Reply:
@niranjan94 Can ember helpers be used instead of it? Because I'm facing a problem of checking for a specific ticket. If I use a helper, I think it would be possible to pass parameters into it as given below. 
`{{#if (donation-valid ticket.minPrice ticket.maxPrice) }}`

Reply:
![Screenshot 2019-07-03 at 6 13 02 PM](https://user-images.githubusercontent.com/25197147/60592482-5cee9f00-9dbe-11e9-8643-af65fe280f02.png)
I was able to achieve this via ember helpers.